Tools/CISupport/ews-build/steps.py

```diff
@@ -53,9 +53,11 @@
 BUG_SERVER_URL = 'https://bugs.webkit.org/'
 COMMITS_INFO_URL = 'https://commits.webkit.org/'
 S3URL = 'https://s3-us-west-2.amazonaws.com/'
+S3_BUCKET = f'ews-archives.webkit{custom_suffix}.org'
 S3_RESULTS_URL = f'https://ews-build{custom_suffix}.s3-us-west-2.amazonaws.com/'
 CURRENT_HOSTNAME = socket.gethostname().strip()
 EWS_BUILD_HOSTNAME = 'ews-build.webkit.org'
+EWS_BUILD_UAT_HOSTNAME = 'ews-build.webkit-uat.org'
```

Comment:
Add EWS_BUILD_DEV_HOSTNAME too.

Reply:
Probably should change these to `EWS_PRODUCTION_HOSTNAMES` and `EWS_UAT_HOSTNAMES`, does that sound like a good plan? cc @lingcherd